example/lib/main.dart

```diff
@@ -44,41 +47,172 @@ class MyHomePage extends StatefulWidget {
+        color: Colors.red,
+      ),
+      thisMonthDayBorderColor: Colors.grey,
+      weekFormat: false,
+      markedDatesMap: _markedDateMap,
+      height: 420.0,
+      selectedDateTime: _currentDate2,
+      customGridViewPhysics: NeverScrollableScrollPhysics(),
+      markedDateShowIcon: true,
+      markedDateIconMaxShown: 2,
+      markedDateMoreShowTotal: false, // null for not showing hidden events indicator
+      showHeader: false,
+      minSelectedDate: _currentDate,
+      maxSelectedDate: _currentDate.add(Duration(days: 60)),
+//      inactiveDateColor: Colors.black12,
+      onCalendarChanged: (DateTime date){
+        this.setState(() => _currentMonth = DateFormat.yMMM().format(date));
+      },
+    );
+
+
+    return new Scaffold(
+      appBar: new AppBar(
+        title: new Text(widget.title),
+      ),
+      body: SingleChildScrollView(
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          mainAxisAlignment: MainAxisAlignment.start,
+          children: <Widget>[
+            //custom icon
+            Container(
+              margin: EdgeInsets.symmetric(horizontal: 16.0),
+              child: _calendarCarousel,
+            ), // This trailing comma makes auto-formatting nicer for build methods.
+
+            //custom icon without header
+            Container(
+              margin: EdgeInsets.only(
+                top: 30.0,
+                bottom: 16.0,
+                left: 16.0,
+                right: 16.0,
+              ),
+              child: new Row(
+                children: <Widget>[
+                  Expanded(
+                    child: Text(
+                      _currentMonth,
+                      style: TextStyle(
+                        fontWeight: FontWeight.bold,
+                        fontSize: 24.0,
+                      ),
+                    )
+                  ),
+                  FlatButton(
+                    child: Text('PREV'),
+                    onPressed: (){
+                      setState(() {
+                        _currentDate2 = _currentDate2.subtract(Duration(days: 30));
+                        _currentMonth = DateFormat.yMMM().format(_currentDate2);
+                      });
+                    },
+                  ),
+                  FlatButton(
+                    child: Text('NEXT'),
+                    onPressed: (){
+                      setState(() {
+                        _currentDate2 = _currentDate2.add(Duration(days: 30));
+                        _currentMonth = DateFormat.yMMM().format(_currentDate2);
+                      });
+                    },
+                  )
+                ],
+              ),
+            ),
+            Container(
+              margin: EdgeInsets.symmetric(horizontal: 16.0),
+              child: _calendarCarouselNoHeader,
+            ), //
+          ],
         ),
-      ), // This trailing comma makes auto-formatting nicer for build methods.
+      )
```

Comment:
Why don't you like the trailing comma?

Reply:
Oh, sorry for that, I'm not hating a trailing comma, but I think it got deleted when I'm restructured the sample code.